lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -498,6 +498,9 @@ def detect_es_major_version
       @_es_info ||= client.info
       begin
         unless version = @_es_info.dig("version", "number")
+          if version != @default_elasticsearch_version
+            log.warn "Elasticsearch version glitch exists. Actual #{version}, Expected: #{@default_elasticsearch_version}"
+          end
```

Comment:
Could you add test code for this?

Reply:
I re-consider this check and removed from plugin code in https://github.com/uken/fluent-plugin-elasticsearch/pull/913/commits/39353bc01acb3fc03ed412b291e87c7263d20598.